### DISCUSSION TITLE
Connecting to a local MS sql server using integrated security

### DISCUSSION THREAD

**Discussion by GGAlanSmithee** [prisma/prisma#11492] [Q&A] [answered]
I can't find an example of a connection string to a MS SQL Server DB using integrated security. Is this possible? Here's what I've tried so far:

`DATABASE_URL="sqlserver://localhost:1434;database={DB_NAME};integratedSecurity=true;trustServerCertificate=true;"`

as per how you would do it in .NET EF for example.

The error I'm getting when running `prisma db pull` is:

`Error: Error parsing connection string: Conversion error: Invalid JDBC token in database URL`

Thanks in advance!

**Comment by Jolg42** [comment, marked as answer]
Hi @GGAlanSmithee !

Could you try to escape any special character like mentioned in the SQL server docs ('=', ';', '[]', space and '{}')? 
https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15#escaping-values-in-the-connection-url

I think the error might be around parsing `{DB_NAME}` here.

**Reply by Jolg42** [reply]
You might also want to try this 
```
sqlserver://localhost:1433;initialCatalog=sample;integratedSecurity=true;trustServerCertificate=true;
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
